src/main/java/frc/robot/vision/UpdatePose.java

```diff
@@ -9,6 +9,7 @@ public class UpdatePose extends Command{
   private final DriveSubsystem drive;
 
   public UpdatePose(VisionSubsystem vision, DriveSubsystem drive){
+    addRequirements(vision, drive);
```

Comment:
I'd say that this should not require the drive subsystem. This isn't really using the drive subsystem, and if it did require it then we would be able to either drive or update the pose, not both at the same time.